core/Controller/LostController.php

```diff
@@ -312,16 +313,14 @@ public function setPassword($token, $userId, $password, $proceed) {
 
 	/**
 	 * @param string $input
-	 * @throws \Exception
+	 * @throws ResetPasswordException
```

Comment:
```suggestion
	 * @throws ResetPasswordException
	 * @throws \OCP\PreConditionNotMetException
```

`setUserValue` can throw a `PreConditionNotMetException`.